lib/galaxy/datatypes/hubAssembly.py

```diff
@@ -0,0 +1,59 @@
+"""
+HubAssembly datatype
+"""
+import logging
+import galaxy.version as version
+
+# Support for Galaxy <= 16.01
+if version.VERSION_MAJOR <= "16.01":
+    from galaxy.datatypes.images import Html
+else:
+    from galaxy.datatypes.text import Html
```

Comment:
You don't need to care for older Galaxy versions, since this PR will be merged in the `dev` branch and the datatype released only in Galaxy 16.07.